<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisCODe wizard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget[objectName=&quot;Form&quot;] {
	background-color: palette(light);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true"> QTabWidget#tabWidget::tab-bar {

 }

QTabWidget[objectName=&quot;tabWidget&quot;]&gt;QTabBar::tab {
    border: 1px solid #C4C4C3;
    border-bottom-color: palette(dark);
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    padding: 3px 6px;
    margin-left: 9px;
	margin-right: -6px;
	background-color: palette(button);
	color: palette(shadow);
}

QTabWidget[objectName=&quot;tabWidget&quot;]&gt;QTabBar::tab:selected {
    background-color: palette(button);
    border-bottom-style: none;
	color: palette(text);
}

QTabWidget[objectName=&quot;tabWidget&quot;]&gt;QWidget {
	background-color: palette(light);
	border-bottom: 1px solid palette(dark);
}

QTabWidget[objectName=&quot;tabWidget&quot;]&gt;QWidget&gt;QWidget {
	background-color: palette(button);
	border-bottom: 1px solid palette(dark);
}</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <property name="toolTip">
       <string/>
      </property>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5" columnminimumwidth="100,0,0">
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>D&amp;ependencies</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="buddy">
          <cstring>lstDeps</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;DCL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>cbDCL</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edName"/>
       </item>
       <item row="3" column="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnAddDep">
           <property name="toolTip">
            <string>Add new library to dependencies</string>
           </property>
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveDep">
           <property name="toolTip">
            <string>Remove selected dependency</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="edAuthor"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&amp;Author</string>
         </property>
         <property name="buddy">
          <cstring>edAuthor</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbDCL">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>edName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnBrowseDCL">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QTextEdit" name="edDesc">
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>De&amp;scription</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="buddy">
          <cstring>edDesc</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QListWidget" name="lstDeps"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="toolTip">
       <string/>
      </property>
      <attribute name="title">
       <string>Streams</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" columnminimumwidth="100,0,0">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Input</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTableWidget" name="tblStreamIn">
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Data type</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAddStreamIn">
           <property name="toolTip">
            <string>Add new input stream</string>
           </property>
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveStreamIn">
           <property name="toolTip">
            <string>Remove selected stream</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Output</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTableWidget" name="tblStreamOut">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Data type</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="btnAddStreamOut">
           <property name="toolTip">
            <string>Add new output stream</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveStreamOut">
           <property name="toolTip">
            <string>Remove selected stream</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <property name="toolTip">
       <string/>
      </property>
      <attribute name="title">
       <string>Handlers</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" columnminimumwidth="100,0,0">
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="rdAct">
           <property name="text">
            <string>Active</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdPas">
           <property name="text">
            <string>Passive</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rdTri">
           <property name="text">
            <string>Triggered</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Handlers</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btnAddHandler">
           <property name="toolTip">
            <string>Add new handler</string>
           </property>
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveHandler">
           <property name="toolTip">
            <string>Remove selected handler</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHandlerUp">
           <property name="toolTip">
            <string>Move selected handler up</string>
           </property>
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHandlerDown">
           <property name="toolTip">
            <string>Move selected handler down</string>
           </property>
           <property name="text">
            <string>Down</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Depends on</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QListWidget" name="lstHandlers"/>
       </item>
       <item row="5" column="1">
        <widget class="QListWidget" name="lstHandDeps"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <property name="toolTip">
       <string/>
      </property>
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" columnminimumwidth="100,0,0">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Properties</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTableWidget" name="tblProps">
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>60</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Data type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Default value</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Display type</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="btnAddProp">
           <property name="toolTip">
            <string>Add new property</string>
           </property>
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveProp">
           <property name="toolTip">
            <string>Remove selected property</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Preview</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextEdit" name="teCpp"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="toolTip">
        <string>Close Wizard without saving changes</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGenerate">
       <property name="toolTip">
        <string>Generate new component</string>
       </property>
       <property name="text">
        <string>&amp;Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create new DisCODe component...</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lblInfoIcon">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblInfo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cbDCL</tabstop>
  <tabstop>btnBrowseDCL</tabstop>
  <tabstop>edName</tabstop>
  <tabstop>btnAddDep</tabstop>
  <tabstop>btnRemoveDep</tabstop>
  <tabstop>edAuthor</tabstop>
  <tabstop>edDesc</tabstop>
  <tabstop>tblStreamIn</tabstop>
  <tabstop>btnAddStreamIn</tabstop>
  <tabstop>btnRemoveStreamIn</tabstop>
  <tabstop>tblStreamOut</tabstop>
  <tabstop>btnAddStreamOut</tabstop>
  <tabstop>btnRemoveStreamOut</tabstop>
  <tabstop>btnAddHandler</tabstop>
  <tabstop>btnRemoveHandler</tabstop>
  <tabstop>btnHandlerUp</tabstop>
  <tabstop>btnHandlerDown</tabstop>
  <tabstop>tblProps</tabstop>
  <tabstop>btnAddProp</tabstop>
  <tabstop>btnRemoveProp</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnGenerate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
